packages/app/templates/ui-extensions/projects/checkout_ui/src/index.liquid

```diff
@@ -1,3 +1,4 @@
+{%- if flavor == "react" -%}
```

Comment:
Yeah, `recursiveDirectoryCopy` made this a bit tricky.  When we copied the template, the path to the partials changed, so the partials no longer worked.  We could solve this, but the partials seemed like overkill anyway.  Only the `.toml` file had truly shared code (and only 2 properties at that), so we took the opportunity to simplify things a bit, rather than adding complexity into the CLI around complexity in the templates that wasn't really needed.